drivers/serial/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ zephyr_library_sources_if_kconfig(uart_esp32.c)
 zephyr_library_sources_if_kconfig(uart_sifive.c)
 zephyr_library_sources_if_kconfig(uart_gecko.c)
 zephyr_library_sources_if_kconfig(leuart_gecko.c)
+zephyr_library_sources_ifdef(CONFIG_UART_LPC11U6X uart_lpc11u6x.c)
```

Comment:
Why not use `zephyr_library_sources_if_kconfig()`?

Reply:
My bad, we should use it. It is fixed now.